### PR TITLE
updated profile

### PR DESCRIPTION
**Pull request by Shalini469717**
added socials, updated bio, added blog link

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6518"></a>

### CODE REVIEW

File: data/Shalini469717.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "Shalini Muskula",
   "type": "personal",
-  "bio": "Web Developer",
+  "bio": "Web Developer | Open Source | Blogger",
   "links": [
     {
       "name": "Twitter",
```
Comment (GauravKanwat) [inline data/Shalini469717.json:7]: Please add a link for twitter and close with proper format.

**Review by GauravKanwat — CHANGES_REQUESTED** [review]
You can check the code you typed on "jsonlint.com" for any errors. :+1:

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
The package lock file should only be committed if the package.json file has changes. There could be two reasons why the lock file has been updated for you:

1. You used `npm install` instead of `npm ci` or
2. You have GitHub Actions running on your fork. This should be disabled by default, please do not enable.

Please undo any changes to the package lock file. 

Here is a video that will show you how to do it: https://youtu.be/jHkIXYVpMTo

**Comment by Shalini469717** [comment]
Thanks for the review, I was using Linkfree cli and used `npm install`.
Hey, maybe we add `npm ci` to Linkfree cli, since I found it confusing what to use while installing dependencies.

**Comment by eddiejaoude** [comment]
> Thanks for the review, I was using Linkfree cli and used `npm install`.
> Hey, maybe we add `npm ci` to Linkfree cli, since I found it confusing what to use while installing dependencies.

Sorry @Shalini469717 for the confusion, we have [docs](https://linkfree.io/docs/environments/linkfree-cli) on using the CLI but I think we are missing the `npm ci` step and most people do `npm install` which causes this problem. I will add these steps to our docs now

Was the video clear on how to remove the changes to the package lock file? Any questions let us know

**Comment by Shalini469717** [comment]
Yeah the video was clear, but the issue I faced was after committing, the package-lock.json file had changed in my local repo's main branch as well. So, it took a bit of time to figure out how to fetch from remote repo and merging it with local repo. 
`$ git checkout EddieLinkFree/main -- package-lock.json`
After that I did everything according to the video and it worked.

**Comment by Shalini469717** [comment]
Yeah will start using branches from now on! Learnt new things again. Thanks for the help!

**Comment by eddiejaoude** [comment]
> Yeah will start using branches from now on! Learnt new things again. Thanks for the help!

we are all always learning, part of the fun of being in tech 🤓

**Review by SaraJaoude — APPROVED** [review]

**Comment by SaraJaoude** [comment]
looks good

**Pull request merged**